Mathlib/Analysis/NormedSpace/OperatorNorm.lean

```diff
@@ -890,11 +891,12 @@ theorem apply_apply' (v : E) (f : E →SL[σ₁₂] F) : apply' F σ₁₂ v f =
 
 variable (𝕜 Fₗ)
 
+-- Note: compilation runs away #7103
```

Comment:
Can you use the same comments as in #7265 to make them easier to grep for? The comment there was
```
-- `noncomputable` is a performance workaround for mathlib4#7103
```